src/server/worldserver/worldserver.conf.dist

```diff
@@ -4147,3 +4147,26 @@ Metric.OverallStatusInterval = 1
 
 #
 ###################################################################################################
+
+###################################################################################################
+#Paymaster#
+#################################
+#
+#    Paymaster
+#        Enable this for a PTR style Paymaster implementation
+#
+#    Paymaster.Announce.Enable
+#         Description: Announces Module when player logs in.
+#         Default:    0 - (Disabled)
+#                     1 - (Enabled)
+
+Paymaster.Announce.Enable = 1
+
+#    Paymaster.AddGold
+#         Configurable gold added amount from Paymaster
+#         Default:    5000
+
+Paymaster.AddGold = 5000
+
+#
+###################################################################################################
```

Comment:
line break preferrably, but other then that, this is really great stuff. Great job.

Reply:
yes
![image](https://user-images.githubusercontent.com/16887899/178769199-e6a68b65-7996-4629-8502-dcd4b899c9b5.png)
I believe it deals with the code style of the project. but it isnt anything project breaking.

Reply:
Yup, updated and finally fixed all the other random whitespace that appeared...seems like the cmake generation for visual studio isn't copying the editor config to the right spot, might have to look at that